phoenix-core/src/main/java/org/apache/phoenix/compile/ExpressionProjector.java

```diff
@@ -39,14 +39,24 @@ public class ExpressionProjector implements ColumnProjector {
     private final Expression expression;
     private final String tableName;
     private final boolean isCaseSensitive;
-    
+    private final String label;
+
+    public ExpressionProjector(String name, String tableName, Expression expression, boolean isCaseSensitive, String label) {
```

Comment:
nit: name and label are strongly related, I'd prefer if they were consecutive parameters.